dinghy-lib/src/errors.rs

```diff
@@ -3,6 +3,7 @@ error_chain! {
     foreign_links {
         Io(::std::io::Error);
         StringFromUtf8(::std::string::FromUtf8Error);
+        ParseIntError(std::num::ParseIntError);
```

Comment:
Bit of nitpicking: our convention is too strip the std:: prefix and Error suffix, so that would be ParseInt.